bsp/nrf5x/nrf52840/board/sdk_config.h

```diff
@@ -3224,7 +3224,7 @@
 // <e> NRFX_SAADC_ENABLED - nrfx_saadc - SAADC peripheral driver
 //==========================================================
 #ifndef NRFX_SAADC_ENABLED
-#define NRFX_SAADC_ENABLED 0
```

Comment:
Please to see the pwm enable , to define the sdk_config.h. No need change the sdk_config.h, just add the config on the Kconfig